File: sentimental-mario/mario.py
# import get_int from cs50
from cs50 import get_int

# prompt user for hight of pyramid between 1 - 8.
# while true check if value is between 1 - 8.
while True:
    height = get_int("Height: ")
    # if ture break, else prompt again
    if height > 0 and height < 9:
        break

# in a loop go through each line and print the appropirate number of # with a space of 2 between the pyramids.
for row in range(height):
    print(" " * (height-row-1), end="")
    print("#" * (row+1), end="")
    print("  ", end="")
    print("#" * (row+1), end="")
    print()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
   int h;
   do
   {
        h = get_int("Height: ");
   }
   while (h < 1 || h > 8);

   for(int i = 0; i < h; i++)
   {

        for(int l = i + 1; l < h; l++)
        {
            printf(" ");
        }
       for(int j = h - 1 - i ; j < h; j++)
        {
            printf("#");
        }
        printf("\n");
   }
}
